be/src/runtime/data_stream_mgr.cpp

```diff
@@ -151,6 +151,8 @@ Status DataStreamMgr::transmit_chunk(const PTransmitChunkParams& request, ::goog
         // in acquiring _lock.
         // TODO: Rethink the lifecycle of DataStreamRecvr to distinguish
         // errors from receiver-initiated teardowns.
+        LOG(WARNING) << request.sender_id() << " sender transmits chunks to a non-existing receiver fragment "
```

Comment:
prefer VLOG

Reply:
help us to know the exchange receiver is alive longer than sender.